src/main/java/mods/eln/Eln.java

```diff
@@ -1148,134 +1165,161 @@ private void registerElectricalCable(int id) {
+            4d));
+        List<String> renderlist = new ArrayList<>(Arrays.asList(
+            "sprites/cable.png",
+            "sprites/cableshoddy.png",
+            "sprites/cablequality.png",
+            "sprites/cableperfect.png",
+            "sprites/cableVHV.png",
+            "sprites/cableVHVshoddy.png",
+            "sprites/cableVHVquality.png",
+            "sprites/cableVHVperfect.png"));
+        List<Float> rendercablesizes = new ArrayList<>(Arrays.asList(
+            1.95f,
+            0.95f,
+            2.95f,
+            0.95f,
+            3.95f,
+            1.95f));
+
+        int renderspriteindex = 0;
+        int renderfloatindex = 0;
+        subId = 3;
+        for(int cablevoltage = 0; cablevoltage <=3; cablevoltage++){
+        for(int cablequality = 0; cablequality <=3; cablequality++) {
+            //while (cablevoltage <= 3) {
+            renderspriteindex = 0;
+            subId++;
+            name = TR_NAME(Type.NONE, qualitylist.get(cablequality) + voltagelist.get(cablevoltage) + "Voltage Cable");
+
+            renderspriteindex = cablequality;
+
+            switch (cablevoltage) {
+                case 0:
+                    power = LVP();
+                    nominalcablevoltage = LVU;
+                    renderfloatindex = 0;
+                    break;
+                case 1:
+                    power = MVP();
+                    nominalcablevoltage = MVU;
+                    renderfloatindex = 2;
+                    break;
+                case 2:
+                    power = HVP();
+                    nominalcablevoltage = HVU;
+                    renderfloatindex = 4;
+                    break;
+                case 3:
+                    power = VVP();
+                    nominalcablevoltage = VVU;
+                    renderfloatindex = 4;
+                    renderspriteindex = renderspriteindex + 4;
+                    break;
+            }
+            activerendermodel = new CableRenderDescriptor("eln", renderlist.get(renderspriteindex), rendercablesizes.get(renderfloatindex), rendercablesizes.get(renderfloatindex + 1));
+            desc = new ElectricalCableDescriptor(name, activerendermodel, "Haxors was Here", false);
+            nominalcablepower = power * valuelist.get(cablequality);
+            switch (cablequality) {
+                case 0:
+                    switch (cablevoltage) {
+                        case 0:
+                            lowVoltageCableDescriptor = desc;
+                            break;
+                        case 1:
+                            meduimVoltageCableDescriptor = desc;
+                            break;
+                        case 2:
+                            highVoltageCableDescriptor = desc;
+                            break;
+                        case 3:
+                            veryHighVoltageCableDescriptor = desc;
+                            break;
+                    }
+                case 1:
```

Comment:
Fair warning, I'll probably rewrite all of this in the future. Because it could be much cleaner... in Kotlin. :P